Explorer/Assets/DCL/Character/CharacterMotion/Systems/CalculateCharacterVelocitySystem.cs

```diff
@@ -117,7 +121,20 @@ private void ResolveVelocity(
             in MovementInputComponent movementInput)
         {
             // Apply velocity based on input
-            ApplyCharacterMovementVelocity.Execute(settings, ref rigidTransform, in cameraComponent, in movementInput, dt);
+#if UNITY_EDITOR
```

Comment:
I would not pollute this function this way: the only thing you need from `Camera` is its `Transform`:
- Change argument to transform
- Separate the logic of this function from query in a simple method
- Introduce 2 queries:
   - one will be responsible for the default behavior and will pass `CameraComponent.transform` with `None[typeof(RandomAvatar)]` filter
   - another one will pass `characterController.ttransform` itself and will have `All[typeof(RandomAvatar)]` filter
   - It will be the most proper way to do it with ECS without modifying the original logic